services/php-apache/Dockerfile

```diff
@@ -1,5 +1,7 @@
-ENV COMPOSER_NO_INTERACTION=1 \
-    DOCROOT=/var/www/html
+ENV COMPOSER_DISCARD_CHANGES=true
+ENV COMPOSER_NO_INTERACTION=true
+ENV COMPOSER_ALLOW_SUPERUSER=true
+ENV DOCROOT=/var/www/html
```

Comment:
What's this for? Is the idea to save having to cd into your docroot? I think it's more straight forward if by default commands run inside the git checkout unless something in the config.yml changes that.

Reply:
Tugboat uses this.

Reply:
Oh! It's _not_ a composer env var. Perhaps docs, otherwise carry on.